docs/examples/color_space_conversion.ipynb

```diff
@@ -18,10 +18,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/usr/local/lib/python2.7/dist-packages/nvidia/dali/backend.py:15: FutureWarning: pybind11-bound class 'nvidia.dali.backend_impl.TensorCPU' is using an old-style placement-new '__init__' which has been deprecated. See the upgrade guide in pybind11's docs. This message is only visible when compiled in debug mode.\n",
```

Comment:
Please remove these messages: they are printed when `backend_impl` is compiled in Debug mode

Reply:
(I think opening the `.ipynb` in a text editor and removing `"outputs"` manually will do it here)

Reply:
Debug build. I will clean that up.

Reply:
Done